include/swift/AST/SourceFile.h

```diff
@@ -14,11 +14,13 @@
 #define SWIFT_AST_SOURCEFILE_H
 
 #include "swift/AST/FileUnit.h"
+#include "swift/AST/SynthesizedFileUnit.h"
 #include "swift/Basic/Debug.h"
 
 namespace swift {
 
 class PersistentParserState;
+class SynthesizedFileUnit;
```

Comment:
This line can be removed. I'll remove it in a follow-up PR, unless I need to update this one.

Reply:
Done in https://github.com/apple/swift/pull/30917.